app/frontend/src/services/index.js

```diff
@@ -1,2 +1,3 @@
+export { default as emailService } from './emailService';
 export { default as helloService } from './helloService';
```

Comment:
Do we still need this service?

Reply:
I knew I forgot something. Thanks for the catch